.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -4,6 +4,7 @@
 <!-- Add the issue number that is fixed by this PR (In the form Fixes #123) -->
 
 Fixes #
+[Add issue number here. If you do not solve the issue entirely, please change the message e.g. "First steps for issues #IssueNumber]
```

Comment:
This is not needed as already at line 4 description about it is given. You can edit that.

Reply:
Both lines have different meanings, please see carefully.

Reply:
That's why I am saying that please edit at line 4 to merge both information. As both are related to issue number.